css/styles.css

```diff
@@ -59,29 +162,58 @@ a {
 
 section {
   text-align: center;
-  margin: 6.5em auto;
+  margin: 80px auto;
+}
+
+h1,
+h2,
+h3,
+p {
+  color: rgba(23, 43, 77, 1);
 }
 
 h1 {
   font-size: 2.1em;
   font-weight: bold;
+  margin-top: 3em;
 }
 
 span {
-  color: #36b37e;
   display: block;
 }
 
+button {
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 15px auto;
+}
```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that your button design matches the given design as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=rPm74ST0zW6b1ScP-0) matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

![S btn](https://github.com/flov3rh/portfolio/assets/43528594/d0caf26c-f624-454a-af59-99df81eb59c4)

</td>
      <td>

![F btn](https://github.com/flov3rh/portfolio/assets/43528594/320959f5-b9ad-46ca-85f7-54f82bc8b7d2)

</td>
    </tr>
  </table>



- [ ] And kindly follow the button styling: `Active, Hover and Pressed`

![H](https://github.com/flov3rh/portfolio/assets/43528594/5e9a201a-3420-499a-845e-609c06adeaa2)

Reply:
Added :active effect, however, I do not see any difference between enabled and on hover